.github/workflows/pr.yaml

```
@@ -94,6 +96,12 @@ jobs:
     uses: rapidsai/shared-workflows/.github/workflows/checks.yaml@branch-24.12
     with:
       enable_check_generated_files: false
+  clang_tidy:
```

Name this with a dash, for consistency.
```suggestion
  clang-tidy:
```